### ISSUE TITLE
[a11y] [WCAG 2.4.1]: The page doesn't offer a way to skip repetitive content.

### ISSUE THREAD

**Issue by dkatzz** [civiform/civiform#9916]
OBSERVED RESULTS:
--
When navigating the TestCity CiviForm page using the keyboard, the header repeats on every product page. Users have to tab through all the icons in the header, with no option to skip them. This makes navigation slower and more cumbersome as there is no "Skip to main content" button to bypass the repetitive elements.

USER IMPACT:
--
The Screen reader users may experience difficulty navigating through repetitive content, such as menus or headers, if they cannot easily skip past these blocks to access the main content.

EXPECTED RESULTS:
--
When navigating the TestCity CiviForm page using the keyboard, users should be able to bypass the repetitive header elements and directly access the main content. This can be achieved by providing a "Skip to main content" button, allowing for a smoother and more efficient navigation experience for keyboard users.

STEPS TO REPRODUCE:
--
0. Enable ChromeVox using "CONTROL+ALT+Z".
1. Go to [TestCity CiviForm](https://ns.civiform.dev/programs)
2. Navigate through the page with the keyboard and notice that there is no method to skip the repetitive header content.

SCREENCAST:
--
- Chromebook+ChromeVox:[Evidence](https://drive.google.com/file/d/1lEm1SagPMTnjyrFeZjrJRy-onP7abCA6/view?usp=drive_link&resourcekey=0-LXShg_3qnWDpCjBXa4UlOw)
- Windows+NVDA:[Evidence](https://drive.google.com/file/d/1hlxMRPrZAyE_ByZkAtQqM2P9CUsk_vg-/view?usp=drive_link&resourcekey=0-FgKcm-SLZ7dk7RoYooWKsQ)

ADDITIONAL INFORMATION:
--
**WCAG rule:**

[WCAG 2.4.1](https://www.w3.org/TR/WCAG22/#bypass-blocks)

**Enviroment details:**

Machine: Chromebook Version 13.3

Browser: Chrome (132.0.6834.208 (Official Build) (64-bit))

Machine: Windows 11 Enterprise 23H2

Screen reader: ChromeVox, NVDA

Bug Filling Time: 20 mins

**Comment by gwendolyngoetz** [comment]
Related: https://github.com/civiform/civiform/issues/6067
